### PR TITLE
Updated Step History section of the reference documentation.

### PR DESCRIPTION
**Pull request by cppwfs**
It now contains definitions for the attributes of the metrics that are captured.
Also adds defintions for the metrics themselvess

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc [hunk 1]
```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
 [[dashboard-job-executions-steps-progress]]
 === Step Execution History
 
-Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others.
+Under *Step Execution History*, you can also view various metrics associated with the selected step, such as duration, read counts, write counts, and others across all of its executions.
+For each metric there are 5 attributes that are available for each metric:
```
Comment (onobc) [inline spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:463]: **[NIT]** Redundant "For each metric..... for each metric". I think we can drop the trailing "for each metric:".

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc [hunk 2]
```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
...
+
+* Count - The number of step executions that the metric could have participated.  It is not a count for the number of times the event occurred during each step execution.
+* Min - Shows the minimum value for this metric across all the step executions.
```
Comment (onobc) [inline spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:466]: I think we can drop the "Shows the " from the descriptions.

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc [hunk 3]
```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
...
+* Max - Shows the maximum value for this metric across all the step executions.
+* Mean - Shows the mean value for this metric across all the step executions.
+* Max - Shows the standard deviation for this metric across all the step executions.
+
+NOTE: Counts (commit count for example) for a specific step execution, can be obtained by retrieving the job execution detail.
+
+The Step Execution contains the following metrics:
```
Comment (onobc) [inline spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:473]: Is `job execution detail`(line above)  the same as `step execution` ?

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc [hunk 4]
```diff
@@ -459,7 +459,29 @@ When that happens, check the server log files for further details.
...
+
+* Commit Count - Attributes for the number of commits of all the executions for the given step.
+* Duration - Attributes for the duration of all the executions for the given step.
```
Comment (onobc) [inline spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:476]: What is `Attributes for ` in this context?

**Review by onobc — CHANGES_REQUESTED** [review]
@cppwfs thanks for adding this - it will help clarify the fields in the UI. I have a couple of nit-level comments and a couple of question/clarifications. Other than that, LGTM.

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc (hunk 1 above)
Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:463]: Doh!   Good catch.

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc (hunk 3 above)
Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc:473]: Removing this line because this information is covered in another section.

**Comment by cppwfs** [comment]
@onobc updated based on your comments.   Thanks for the review!

**Comment by onobc** [comment]
Closed via https://github.com/spring-cloud/spring-cloud-dataflow/commit/3d1eebdef2892474929e114e997b86accaa8d54c

**Pull request merged**